### DISCUSSION TITLE
THE DOM

### DISCUSSION THREAD

**Discussion by Goldenboy3023** [Goldenboy3023/Hello-World-#6] [Q&A] [answered]
THE DOM how do I select an element.

**Comment by onthegocode** [comment, marked as answer]
To select an Element from the DOM all you have to do is use a `document.querySelector("")` and inside the () either put the element name if you want to select the element, but if you want to select a class or id you need to use .class or #id. Also to let you know querySelector only selects 1 element if you want to select a list of elements you need to use querySelectorAll.

**Reply by Goldenboy3023** [reply]
ok
